src/init/inst-functions.sh

```diff
@@ -1156,6 +1156,14 @@ InstallAgent()
 
     ${INSTALL} -d -m 0750 -o root -g ${WAZUH_GROUP} ${INSTALLDIR}/wodles/azure
     ${INSTALL} -m 0750 -o root -g ${WAZUH_GROUP} ../wodles/azure/azure-logs.py ${INSTALLDIR}/wodles/azure/azure-logs
+
+    # Install debugging symbols if available
+    mkdir -p 
```

Comment:
Hi @RamiroRD 

I think you wanted to create a directory ${INSTALLDIR}/.symbols. Am I right?
 
 mkdir -p ${INSTALLDIR}/.symbols

Reply:
That's correct, thanks!